test/distributions/bharshbarg/fastFollowerEquality.good

```diff
@@ -1,3 +1,5 @@
+warning: The variable 'dataParTasksPerLocale' is unstable and its interface is subject to change in the future
+note: 'dataParTasksPerLocale' was set via a compiler flag
```

Comment:
I was surprised to see this impact tests without `--warn-unstable`, but I think that's because the config is getting set at execution time and `--warn-unstable` is a compilation time flag (so it's harder to check at execution time).  If you want to take a look into altering that, I won't stop you, but it would also be okay to open an issue about it and I'll try to take a look this sprint

Reply:
On a quick look, it looks like [this PR](https://github.com/chapel-lang/chapel/pull/22849) wired the `--warn-unstable` into the runtime and specifically Engin marked [this commit ](https://github.com/chapel-lang/chapel/pull/22849/commits/2122e524bf664c1f245b16a75e21934b272b7e62). The logic seems correct to me and I'm not sure I know enough to untangle what's going on there. 
I will create an issue for this.

Reply:
Weeeeeird